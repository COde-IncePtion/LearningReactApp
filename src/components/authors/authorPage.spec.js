import React from "react";
import AuthorPage from "./authorPage";
import {shallow} from "enzyme";

it("should render authorPage component", () => {
    let component = shallow(<AuthorPage/>);
    expect(component.instance().state).toEqual({authors: []});
});

it("should component snapshot", () => {
    let component = shallow(<AuthorPage/>);

    expect(component).toMatchSnapshot();
});