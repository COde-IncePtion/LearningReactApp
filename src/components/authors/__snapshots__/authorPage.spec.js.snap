// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should component snapshot 1`] = `
<div>
  <h1>
    Authors
  </h1>
  <Link
    activeClassName="active"
    className="btn btn-primary"
    to="addAuthor"
  >
    Add author
  </Link>
  <AuthorsList
    authors={Array []}
  />
</div>
`;
